.github/workflows/release-operator.yml

```diff
@@ -93,6 +93,7 @@ jobs:
 
           # Create Distribution Matrix
           echo "dist_matrix=$(echo -n "${{ env.BUILD_PLATFORMS }}" | jq -csR '. | split(",")')" >> $GITHUB_OUTPUT
+          echo "dist_matrix_dash=$(echo -n "${{ env.BUILD_PLATFORMS }}" | tr '/' '-' | jq -csR '. | split(",")')" >> $GITHUB_OUTPUT
```

Comment:
yaml properties of `linux/amd64` dont play nice, so have to replace `/` with a `-`